docs/data/toolpad/getting-started/overview.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+<p class="description">
+Learn core concepts of the Toolpad app and you are ready to ship your application in minutes!
+</p>
+
+## What is Toolpad?
+
+Toolpad is a self-hosted low-code internal tools builder designed to extend the MUI React components. It's designed for developers of all trades who want to save time building internal applications. Drag & drop pre-built UI components, connect your data sources, release your app - and you're done!
+
+![Editor screenshot](/static/toolpad/marketing/index-hero-video-poster.jpg)
+
+## How Toolpad works?
+
+- [Build UI](/toolpad/core-concepts/building-ui/) using our drag & drop editor
+- [Configure and fetch](/toolpad/core-concepts/connecting-to-datasources/) data from HTTP endpoints, Google sheets or even directly from PostgreSQL
+- Connect data with UI elements using [binding expressions](/toolpad/core-concepts/data-binding/)
+- When you are ready to share - [deploy](/toolpad/core-concepts/versioning-and-deploying/) a new version
+
+## Navigating Toolpad
+
+![Toolpad app](/static/toolpad/terminology.png)
+
+There are 5 most frequently used areas:
+
+- **Navigation** - return to homepage, preview or deploy app
+- **Explorer** - this is there you define and manage pages, connection configurations, custom component definitions, etc...
+- **Component library** - drag component instance from this drawer and drop it in **Canvas** to build UI
+- **Canvas** - this is a place where you build your app and immediately preview how it looks like
+- **Inspector** - modify properties of a different instance
+
+## What's next?
+
+[Try to build your first app](/toolpad/getting-started/quickstart/)
```

Comment:
This can be cut, as the docs UI already includes a link to the next page at the end.

```suggestion
```